02-peer-protocol.md

```diff
@@ -518,12 +518,51 @@ reached acceptable depth.
    * [`channel_id`:`channel_id`]
    * [`u32`:`funding_feerate_perkw`]
    * [`point`:`funding_pubkey`]
+   * [`splice_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `splice_tlvs`
+2. types:
+    1. type: 0 (`channel_type`)
+    2. data:
+        * [`...*byte`: `type`]
+    1. type: 2 (`verification_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
+    1. type: 4 (`signing_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
+    1. type: 6 (`tx_verification_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
+    1. type: 8 (`tx_signing_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
 
 1. type: 76 (`splice_ack`)
 2. data:
    * [`chain_hash`:`chain_hash`]
    * [`channel_id`:`channel_id`]
    * [`point`:`funding_pubkey`]
+   * [`splice_ack_tlvs`:`tlvs`]
+
+1. `tlv_stream`: `splice_ack_tlvs`
+2. types:
+    1. type: 0 (`verification_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
+    1. type: 2 (`signing_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
+    1. type: 4 (`tx_verification_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
+    1. type: 6 (`tx_signing_musig2_pubnonce`)
+    2. data:
+        * [`66*byte`: `nonces`]
+
+1. type: 78 (`splice_rejected`)
```

Comment:
We've been leaning on tx_abort for canceling in other places. What are the pros of adding a new message type for this?